### PR TITLE
Feat/powerbi popover

### PR DESCRIPTION
**Pull request by Gustav-Eikaas**
# Description

Please include a summary of the change and which issue is fixed.
List any external dependencies that are required for this change.

Completes: AB#FILL_IN_YOUR_ISSUE_ID

image

## Type of change

Remove options that are not relevant.

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Checklist

-   [ ] My code follows the style guidelines of this project
-   [ ] I have performed a self-review of my own code.
-   [ ] I have linked my DevOps task using the AB# tag.
-   [ ] My code is easy to read.
-   [ ] Documentation and comments is added where needed.
-   [ ] My code is covered by tests.
-   [ ] UX has reviewed relevant UI contributions.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,7 +80,7 @@
     "@equinor/fusion-framework-react-app": "^2.0.30",
     "@equinor/fusion-framework-react-module-context": "^1.0.17",
     "@equinor/workspace-ag-grid": "^0.3.5",
-    "@equinor/workspace-fusion": "^0.15.4",
+    "@equinor/workspace-fusion": "^0.16.1",
```
Comment (jaysencpp) [inline package.json:83]: Any breaking changes not related to PBI in this new version?

File: libs/shared/src/lib/hooks/usePBIOptions.tsx [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
 export type Filters = {
   table: string;
   column: string;
 };
-export function usePBIOptions(reportUri: string, filters?: Filters): PowerBiConfig {
-  const { getEmbed, getToken } = usePBIHelpers();
+export function usePBIOptions(
+  reportUri: string,
+  reportName: string,
+  filters?: Filters
```
Comment (jaysencpp) [inline libs/shared/src/lib/hooks/usePBIOptions.tsx:13]: Could you add some TSDOC to the three parameters here

File: libs/shared/src/lib/components/powerbi/ReportMeta.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+  if (!data) {
+    throw new Error();
+  }
+  return (
+    <Popover
+      open
+      anchorEl={anchor}
+      title={`Report information ${reportName}`}
+      placement="bottom"
```
Comment (jaysencpp) [inline libs/shared/src/lib/components/powerbi/ReportMeta.tsx:30]: Do we want to set a width here or just let it be based on the content of the popover?

**Comment by jaysencpp** [comment]
I think you might be missing some PBI reports, could to double check? For example jcainstallation

File: package.json (hunk 1 above)
Comment (Gustav-Eikaas) [reply package.json:83]: Nope

**Comment by Gustav-Eikaas** [comment]
I checked all the references to usePBIOptions. Ill double check

**Comment by jaysencpp** [comment]
Pipeline is failing because I forgot to remove the `^` from nrwl/storybook. Can you just remove that from package.json and to another npm install

File: libs/shared/src/lib/components/powerbi/ReportMeta.tsx (hunk 3 above)
Comment (Gustav-Eikaas) [reply libs/shared/src/lib/components/powerbi/ReportMeta.tsx:26]: I tried letting it be dynamic but I found it to be weird in size

Comment (Gustav-Eikaas) [reply libs/shared/src/lib/components/powerbi/ReportMeta.tsx:26]: Maybe minWidth would be viable

**Review by jaysencpp — APPROVED** [review]

**Pull request merged**